### PR TITLE
Instructions to share /home/vagrant on Windows

### PR DESCRIPTION
**Pull request by neynah**
Full Credit: @paulproteus

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -88,3 +88,50 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
...
+### create mask = 0750
+### directory mask = 0700
+### read only = no
+### valid users = vagrant
+### path = /home/vagrant
+###
+### Then set a password for the vagrant user on Linux and enable it for
+### samba use by running:
+###
+### pwgen -1  # generate a password
```
Comment (zarvox) [inline Vagrantfile]: `pwgen` may not be installed by default; maybe add `pwgen` to the `apt-get install` line above?

**Comment by neynah** [comment]
I added pwgen, thanks.

**Comment by paulproteus** [comment]
+1 @neynah can you add pwgen to the apt-get install line?​

**Comment by neynah** [comment]
Did I add it correctly in 5ce7d15a159ae47e4ccf2e1a13114ff6d44be5da

**Comment by paulproteus** [comment]
Yes! I'll merge now.

**Comment by paulproteus** [comment]
A couple of random notes:

* The fact that you tested these instructions is very important, so you don't have to give me full credit without at _least_ taking credit yourself for the part you did do!

* To update the pull request, I notice you modified the previous commit and did a force push. One thing you could do instead (maybe you already know this) is to make one change on top, and then push the branch again, but without doing force. When I'm reviewing things, I like to see a commit on top in response to a requested change, rather than the whole commit to change. Different reviewers might have different opinions on this.

Anyway, many many thanks for verifying that these instructions work!

**Comment by neynah** [comment]
Ahhh. OK, I will keep that in mind. Thank you. :)

It was a good exercise!

**Pull request merged**